doc/whats_new.rst

```diff
@@ -26,6 +26,7 @@ The following authors contributed for the first time. Thank you so much! 🤩
 
 * `Laetitia Fesselier`_
 * `Jonathan Vanhoecke`_
+* `Pierre Guetschel`_
```

Comment:
please add Adam Li to the list of authors a few lines below ("The following authors had contributed before.")